linux/src/async_socket_linux.c

```diff
@@ -50,6 +54,7 @@ MU_DEFINE_ENUM_STRINGS(ASYNC_SOCKET_SEND_SYNC_RESULT, ASYNC_SOCKET_SEND_SYNC_RES
 #define THREAD_COUNT    1
 
 static volatile_atomic int32_t g_thread_access_cnt;
```

Comment:
>nt; [](http://example.com/codeflow?start=48&length=3)

We should be explicit (or consistent - below we do initialize with 0 explicitly)